.eslintrc

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
Can we figure out how to reference the google style and just override the explicit changes?